cg/models/report/validators.py

```diff
@@ -1,100 +1,77 @@
 import logging
 from datetime import datetime
 from pathlib import Path
-from typing import Union
+from typing import Any, Optional
+
+from pydantic import ValidationInfo
 
 from cg.constants import (
     BALSAMIC_ANALYSIS_TYPE,
     NA_FIELD,
     NO_FIELD,
     PRECISION,
     REPORT_GENDER,
-    REPORT_SUPPORTED_PIPELINES,
     YES_FIELD,
-    Pipeline,
 )
+from cg.constants.constants import Pipeline, PrepCategory
+from cg.constants.subject import Gender
 from cg.models.orders.constants import OrderType
 
 LOG = logging.getLogger(__name__)
 
 
-def validate_empty_field(value: Union[int, str]) -> str:
-    """Formats an empty value to be included in the report as N/A."""
+def get_report_string(value: Any) -> str:
```

Comment:
I guess this one can be removed by adding string coercion as @islean described

Reply:
I'm not sure if we can, because we'll still need to add validators to transform `None` to a report valid empty field `N/A`

The problem is that we could have both `None` and empty values for these field depending on different analysis